packages/apollo-server-lambda/src/__tests__/ApolloServer.test.ts

```diff
@@ -1,7 +1,7 @@
 import http from 'http';
 import request from 'supertest';
 import express from 'express';
-import { createMockServer } from './mockAPIGatewayServer';
+import { createAPIGatewayV2MockServer } from './mockAPIGatewayServer';
```

Comment:
Why not run tests in this file against both too?

Reply:
Great catch, just added the tests and got them passing locally